dev-docs/bidders/easybid.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: bidder
+title: Easybid
+description: Prebid Easybid Bidder Adapter
+pbjs: true
+biddercode: easybid
+gvl_id: 1068
```

Comment:
The GVL shows this code as belonging to a different company. Do you have evidence that easybid is a subsidiary of ItaliaOnline?

Reply:
HI @bretg , https://www.whois.com/whois/easybid.it shows you that the domain easybid.it belongs to ItaliaOnline.